ansible/data-pipeline.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: data-pipeline
```

Comment:
If this is a new group, add it in folder private_repo/ansible/inventory...

Reply:
done